cucumber/api/features/host_factory_rotate_host.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Rotate a host api key using the host factory.
+
+  Background:
+    Given I am the super-user
+    And I successfully PUT "/policies/cucumber/policy/root" with body:
+    """
+    - !policy
+      id: database
+      body:
+        - !layer users
+        - !host-factory
+          id: users
+          layers: [ !layer users ]
+    """
+    And a host factory token for "database/users"
```

Comment:
This line makes it sounds like you also "successfully PUT a toke".  I think the step def matcher should be change from 
```
Given(/^a host factory token(?: for "([^"]*)")?$/) do |host_factory_id|
```
to
```
Given(/^I have a host factory token(?: for "([^"]*)")?$/) do |host_factory_id|
```